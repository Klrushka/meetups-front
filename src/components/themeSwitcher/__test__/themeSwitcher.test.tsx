import React from 'react';
import rendered, { act } from 'react-test-renderer';
import {ThemeSwitcher} from '../index'


describe('Theme swtcher component', () => {
  let component: rendered.ReactTestRenderer;

  it('Meetup component', () => {
    act(() => {
      component = rendered.create(<ThemeSwitcher />);
    });
  
    expect(component.toJSON).toMatchSnapshot()
  });
 
})
