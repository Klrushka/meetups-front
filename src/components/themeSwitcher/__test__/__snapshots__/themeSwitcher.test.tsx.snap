// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Theme swtcher component Meetup component 1`] = `[Function]`;
